composer.json

```diff
@@ -19,6 +19,8 @@
         "phpunit/phpunit": "^5.7.27",
         "symfony/http-foundation": "^3.4 || ^4.0",
         "zendframework/zend-diactoros": "^1.8",
+        "nyholm/psr7-server": "dev-interface",
+        "http-interop/http-factory-diactoros": "dev-master",
```

Comment:
I will release 1.0.0 when the psr for HTTP factories is released as a package (within few days)